components/intro.js

```diff
@@ -4,12 +4,15 @@ import { Parallax } from 'react-scroll-parallax';
 import BricolageNodeNavigator from '../components/bricolage-nodeNavigator'
 
 export default function Intro({projects}) {
+  console.log("intro", projects);
   return (
     <section className="flex-col md:flex-row flex items-center md:justify-between mt-16 mb-16 md:mb-12">
 
       <h1 className="text-6xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
```

Comment:
We shouldn't have the logo and node navigator inside an `<h1>` for two reasons
1. semantically incorrect
2. makes it hard to reason about styles